assets/js/navigation.js

```diff
@@ -56,7 +56,11 @@
 
 		}
 
-		$this.off( 'hover', position );
+		$this.on( 'mouseleave', function() {
+
+			$( this ).children( '.sub-menu' ).first().removeAttr( 'style' );
```

Comment:
@fjarrett I did try that, and it was throwing an undefined error. I went back and re-worked it. It looks like creating a global `$submenu` variable on initialization allowed it to be passed back into the `mouseleave` function without any issues.